tests/unit/hubble/test_helper.py

```diff
@@ -1,3 +1,4 @@
+from unittest.mock import patch
```

Comment:
please avoid using `unittest` mock, use `monkeypatch` directly so that we only use a single testing framework

Reply:
yes, changing